### PR TITLE
mpena-gu

### PR DESCRIPTION
**Pull request by mepg72**

### CODE REVIEW

**Comment by mepg72** [comment]
quitando Numpy

**Review by mepg72 — COMMENTED** [review]
del Numpy library

**Comment by mepg72** [comment]
del Numpy library

File: chall03/mpena-gu.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            for col in reversed(range(col_init, col_end)):
+                snail_result.append(squared_rows[row][col])
+        else:
+             snail_result.append(squared_rows[row][col_end-1])
+    for row in reversed(range (row_init + 1, row_end - 1)):
+        snail_result.append(squared_rows[row][col_init])
+    snail_str = ", ".join(snail_result)
+    return snail_str
+
+if __name__ == "__main__":
```
Comment (anna-dominguez) [inline chall03/mpena-gu.py:61]: It's a good behaviour to write a condition for your main, but you should do a function `main` called in this condition and not write your main directly in it. Like this :
```python
if __name__ == "__main__":
    main()
```

File: chall03/mpena-gu.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    if len(sys.argv) < 2 or not check_argv(sys.argv[1:]):
+        print ("usage: ./mpena-gu.py <1-9 squared_rows...>")
```
Comment (anna-dominguez) [inline chall03/mpena-gu.py:63]: You could do an external function to avoid duplication of code

**Review by anna-dominguez — COMMENTED** [review]
Your code is almost working, I can see that you're writing your error usage when I write a simple argument like "1", but it should work with it too.

File: chall03/mpena-gu.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python
```
Comment (anna-dominguez) [inline chall03/mpena-gu.py:1]: Your shebang must look like `#!/usr/bin/env python3`

**Comment by mepg72** [comment]
Thanks vesta-nna for you correction, I'll solve all topics and deliver again.

**Pull request merged**